cheatsheets/SAML_Security_Cheat_Sheet.md

```diff
@@ -81,7 +85,16 @@ Additional countermeasures considererd should include:
     - Browser State Exposure (6.4.4)
     - Replay (6.4.5)
 
-Need an architectural diagram? The [SAML technical overview](http://www.oasis-open.org/committees/download.php/11511/sstc-saml-tech-overview-2.0-draft-03.pdf) contains the most complete diagrams. For the Web Browser SSO Profile with Redirect/POST bindings refer to the section 4.1.3. In fact, of all the SAML documentation, the technical overview is the most valuable from a high-level perspective.
+Need an architectural diagram? The [SAML technical overview](https://www.oasis-open.org/committees/download.php/11511/sstc-saml-tech-overview-2.0-draft-03.pdf) contains the most complete diagrams. For the Web Browser SSO Profile with Redirect/POST bindings refer to the section 4.1.3. In fact, of all the SAML documentation, the technical overview is the most valuable from a high-level perspective.
+
+# Unsolicited Response (ie. IdP Initiated SSO) Considerations for Service Providers
+
+Unsolicited Response is inherently less secure by design due to the lack of XSRF type protection. However, it is supported by many due to the backwards compatibility feature of SAML 1.1. The general security recommendation is to not support this type of authentication, but if it must be enabled, the following steps (in additional to everything mentioned above) should help you secure this flow:
```

Comment:
We use `CSRF` almost everywhere else in the cheat sheets rather than `XSRF` - could please change that here?